module.json

```diff
@@ -25,7 +25,7 @@
   ],
   "dependencies": {
     "ble": "^2.1.16",
-    "nrf51-sdk": "^1.0.0"
+    "nrf51-sdk": "^2.0.0"
```

Comment:
@LiyouZhou: There seems to be a conflict with this PR, I think the problem is that the required nordic sdk version has already been updated in develop for this module.